src/Table/elements.js

```diff
@@ -1,197 +1,103 @@
-
-  box-shadow: 0 1px 0 0 hsla(0, 0%, 0%, 0.06);
+// Table
+export const TableElement = styled.table`
+  border-collapse: collapse;
 `;
 
-export const Header = styled.div`
-  display: flex;
-  flex-direction: column;
-
-  width: 100%;
-`;
-
-export const Body = styled.div`
-  display: flex;
-  flex-direction: column;
-
-  width: 100%;
+export const Row = styled.tr`
+  border-top: 1px solid ${({ theme: { palette } }) => palette.veryLightBlue};
+  border-bottom: 1px solid ${({ theme: { palette } }) => palette.veryLightBlue};
+  &:last-child {
+    border-bottom: 0;
+  }
 `;
 
-/**
- * Gets the value of flex attribute for a column.
- *
- * If the given width is an integer, this means that cells width are given in weights relative to each other, hence the
- * resulting flex value (in a container with flex direction set to row), should be the weight (the given integer).
- * If the given value is a width as a string (for instance with px or rem), the flex attribute shouldn't be used (because
- * width attribute is used in place), and thus the flex value is set to initial.
- *
- * @param {string} width - The width of the column.
- *
- * @returns {string} - The value of the CSS flex attribute.
- */
-const getCellFlex = width => (width === parseInt(width).toString() ? width : 'initial');
-
-/**
- * Gets the value of width attribute for a column.
- *
- * If the given width is a string, this means that cells width are given in weights relative to each other, hence the
- * resulting width value (in a container with flex direction set to row), should be the weight (the given string).
- * If the given value is a width as an integer, the width attribute shouldn't be used (because
- * width attribute is used in place), and thus the width value is set to initial.
- *
- * @param {string} width - The width of the column.
- *
- * @returns {string} - The value of the CSS width attribute.
- */
-const getCellWidth = width => (width !== parseInt(width).toString() ? width : 'initial');
-
-const cellAlignToJustify = {
-  left: 'flex-start',
-  center: 'center',
-  right: 'flex-end',
-};
-
-const Cell = styled.div`
-  box-sizing: content-box;
-  display: flex;
-  justify-content: ${({ align }) => cellAlignToJustify[align || 'center']};
-  align-items: center;
-
-  height: 6rem;
-  width: ${({ width }) => getCellWidth(width || 'auto')};
-  flex: ${({ width }) => getCellFlex(width || '1')};
+// Table Header
+export const TableHeader = styled.thead`
+  font-size: 1.2rem;
+  font-weight: ${({
+    theme: {
+      fonts: { weight },
+    },
+  }) => weight.thick};
+  line-height: 1.8rem;
 
-  padding: 0 1rem;
+  text-transform: uppercase;
 
-  color: ${({ theme: { palette } }) => palette.mediumGrey};
+  color: ${({ theme: { palette } }) => palette.darkGrey};
 `;
 
-export const HeaderCell = styled(Cell).attrs({
-  data: ({ j }) => `header-cell-${j}`,
+export const TableHeaderCell = styled.th.attrs({
+  align: ({ align }) => `${align}`,
```

Comment:
According to specification the `align` attribute is obsolete use the `text-align` property instead.
See [doc](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/th#attr-align) for further infos